public/jwpltx.js

```diff
@@ -37,10 +10,44 @@ window.jwpltx = window.jwpltx || {};
     oosv: '5',
     sdk: '0',
   };
+
+  // There are anywhere between 1 to 128 quantiles in any given video, 128 is max for every video
+  const MAX_DURATION_IN_QUANTILES = 128;
   // query params instance.
   let uri;
-  // Current time for live streams.
-  let current;
+  // Time watched after last t event was sent
+  let timeWatched = 0;
+  // Last progress of the video stored
+  let lastVp = 0;
+  // Last quantile reached
+  let lastQ;
+  // Seeking state, gets reset after 1 sec of the latest seeked event
+  let isSeeking = null;
```

Comment:
Can you just initialize this to false?
```suggestion
  let isSeeking = false;
```

Reply:
Good catch, thank you